TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+namespace TelCo.ColorCoder
+{
+    /// data type defined to hold the two colors of clor pair
+    /// </summary>
+    internal class ColorPair
+    {
+        internal Color majorColor;
+        internal Color minorColor;
+        public static Color[] colorMapMajor = new Color[] { Color.White, Color.Red, Color.Black, Color.Yellow, Color.Violet };
+        public static Color[] colorMapMinor = new Color[] { Color.Blue, Color.Orange, Color.Green, Color.Brown, Color.SlateGray };
+        public override string ToString()
+        {
+            return string.Format("MajorColor:{0}, MinorColor:{1}", majorColor.Name, minorColor.Name);
+        }
+        public static void Colorcodemanual()
```

Comment:
Hard to guess that the manual-printing functionality would be inside the file called `ColorPair.cs`. Looking at the other files starting with `Get...`, the name of this function should probably start with `Print...`
Alternative is to name by purpose... like ColorTransalator for the Get..s and ReferenceManual for everything related to the manual.